internal/tools/kops/instance_groups.go

```diff
@@ -125,11 +125,17 @@ func (c *Cmd) UpdateMetadata(metadata *model.KopsMetadata) error {
 		c.logger.WithField("kops-metadata-error", warning).Warn("Encountered a kops metadata validation error")
 	}
 
+	clusterData, err := c.GetClusterSpecInfoFromJSON(metadata.Name, "networking")
+	if err != nil {
+		return err
+	}
+
 	metadata.AMI = AMI
 	metadata.MasterInstanceType = masterMachineType
 	metadata.MasterCount = masterIGCount
 	metadata.NodeMinCount = nodeMinCount
 	metadata.NodeMaxCount = nodeMaxCount
+	metadata.Networking = clusterData
```

Comment:
vvvvvvvv minor: Might use better naming, more specific to networking.

Reply:
+1 on that

Reply:
Done